.changelog/unreleased/api-breaking/1995-deprecate-soft-opt-out.md

```diff
@@ -0,0 +1,2 @@
+- Remove soft opt-out feature. 
```

Comment:
Can you please add the backported PR link.

Reply:
Added.